pkg/controller/containersource/reconcile.go

```diff
@@ -98,15 +99,26 @@ func (r *reconciler) Reconcile(ctx context.Context, object runtime.Object) (runt
 			}
 			r.recorder.Eventf(source, corev1.EventTypeNormal, "Deployed", "Created deployment %q", deploy.Name)
 			source.Status.MarkDeploying("Deploying", "Created deployment %s", args.Name)
-		} else {
-			return source, err
+			// Wait for the deployment to get a status
```

Comment:
I don't understand what this comment means, can you clarify.

Reply:
I think it means wait for the deployment to be reconciled at least once.

Reply:
Expanded, PTAL.